packages/core/src/lib/create-prompt.ts

```diff
@@ -16,20 +16,22 @@ type ViewFunction<Value, Config> = (
 
 function getCallSites() {
   const _prepareStackTrace = Error.prepareStackTrace;
+  let result: NodeJS.CallSite[] = [];
   try {
-    let result: NodeJS.CallSite[] = [];
     Error.prepareStackTrace = (_, callSites) => {
       const callSitesWithoutCurrent = callSites.slice(1);
       result = callSitesWithoutCurrent;
       return callSitesWithoutCurrent;
     };
-
     // eslint-disable-next-line @typescript-eslint/no-unused-expressions, unicorn/error-message
```

Comment:
Can we revert these two unrelated lines?

Or any reason or better practice for changing these?

Reply:
That's fine, the hoisting is necessary due to moving the return.